### PR TITLE
added check duration to array reporter

### PR DESCRIPTION
**Pull request by narekps**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.idea/
```
Comment (kbond) [inline .gitignore:1]: Can you remove this change? You should add a [global .gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) for this.

File: Helper/ArrayReporter.php [hunk 2]
```diff
@@ -65,6 +66,7 @@ public function onAfterRun(CheckInterface $check, ResultInterface $result, $chec
             'status' => $status,
             'status_name' => $statusName,
             'service_id' => $checkAlias,
+            'duration' => round(microtime(true) - $this->checkRunTime, 2),
```
Comment (kbond) [inline Helper/ArrayReporter.php:69]:
```suggestion
            'duration' => microtime(true) - $this->checkRunTime,
```

I think we shouldn't make assumptions on the accuracy here.

**Review by kbond — CHANGES_REQUESTED** [review]
Looks good, just a few minor changes.

File: .gitignore (hunk 1 above)
Comment (narekps) [reply .gitignore]: I removed this change.

File: Helper/ArrayReporter.php (hunk 2 above)
Comment (narekps) [reply Helper/ArrayReporter.php]: I agree with you. fixed.

**Pull request merged**